.github/workflows/release-utils.yml

```
@@ -72,10 +103,78 @@ jobs:
+        with:
+          fetch-depth: 0
+
+      - name: Install Crane
+        uses: imjasonh/setup-crane@v0.7
+
+      - name: Run Wait for Images Script
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ISSUE_NUMBER: ${{ github.event.issue.number }}
+          VERSION: ${{ needs.authorize.outputs.version }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:22:6: Double quote to prevent globbing and word splitting

(shellcheck)